### ISSUE TITLE
[Bug] impossible to use - latest nodejs lts installed, components not compatible with react 17.0.2

### ISSUE THREAD

**Issue by starkajs** [creativetimofficial/ct-material-kit-pro-react#86]
### Version
xxx
### Reproduction link
https://www.creative-tim.com/new-issue/material-kit-pro-react
### Operating System
Windows
### Device
Surface Pro, also tried on MSI
### Browser & Version
NA
### Steps to reproduce
npm install from fresh results in not installing at all. Github issue logged in March matches my experience. 
Tried fresh install of CRA and then installing modules manually in the hope of just copying the components but several of the dependences (react-animate-slider, react-datetime) are not compatible with react 17.0.2
I've installed Node LTS
### What is expected?
a working product... following the instructions on your documentation to generate the starting point
### What is actually happening?
unable to get it working as per comments above

---
### Additional comments
While I appreciate some issues when trying to put the components into my project I did not expect not even being able to run the kits out of the box. Please advise how to get this working otherwise I'm afraid I will have to contest the charges through PayPal as the product is not operational.

**Comment by mtigdemir** [comment]
I purchased the same kit yesterday and tried to build it, unfortunately I'm getting the below error. I also tried to remove/upgrade the relevant package however, other dependencies also failing.  

`npm ERR! While resolving: material-kit-pro-react@1.10.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@">= 15.4.1 < 17.0.0-0" from react-animate-on-scroll@2.1.5
npm ERR! node_modules/react-animate-on-scroll
npm ERR!   react-animate-on-scroll@"2.1.5" from the root project
npm ERR!
`

I was using the free version before, I wanted to support the creative team but sorry to say this. Free version is more stable :(
